### DISCUSSION TITLE
2024/05/wu-guan-zhong/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#103] [General]
# 2024/05/wu-guan-zhong/

https://xiangyun.rbind.io/2024/05/wu-guan-zhong/

**Comment by earfanfan** [comment]
其实中华艺术宫上半年有一个林风眠吴冠中的大展，非常毫横地展出了大量画作，我清明去上海专门去看了两次，里面也有关于两位大师生平的一些描述。比较特别的是，还有一面墙还另外选了与吴冠中有过交集的其他画家的名作，我只看出来各有不同的方向和风格，站在不同的顶峰，倒是因为画家名字都很好听，记了名字，比如有李可染、谢稚柳（字写得特好）、沈柔坚。

那个展览的主题叫做“三个思考维度”，展开的内容是：

>因为是性格迴异的独立个体，展览特辟该主题演绎区对二人艺术探索的共性加以梳理，从“中西之调和”（横向空间之维）、“笔墨的转向”（纵向时间之维）、"为人而艺术"(内外精神之维）三个维度，晨现林风眠和吴冠中在各自历史时空下的种种困惑、困难、奋斗和选择，每一次选择都将他们拉向了现实的骨感，但也因其坚持最终炼就了艺术的高度，拥抱外来文化，还是固守纯粹的民族性？随顺时代创新，还是恪守旧有笔墨套路？ 迎合时流时风，还是表达自我自立门户？三个话题分别从中西文化的横向对照，古今笔墨的纵向拉锅，艺术的外在形态双其内在本质的关系来探讨两位艺术家的艺术道路，从民族、时代个性等角度，由外而内深入到艺术本体问题的恩考。

**Reply by earfanfan** [reply]
额，其实我还有些话想要留言，从我的视角看，感觉你一直有些沉浸在历史环境那些残忍故事造成的伤痛中，似乎没有完全从那种影响中走出来（PS当然这并不是什么问题，并不一定非要走出来）。我的描述可能也不准确，大概就像是疫情期间，虽然我自己本来可以活得好好地，可是看到网络上别的新闻比如上海封城那段时间的各种负面新闻，心理上也不知何故受到了影响，虽未经历却在想象上在感受着。过去当然是会影响现在的，很多当下的社会问题、经济问题如果不停地寻根溯源的话，会发现最初的问题在很久以前。环境当然也是会影响人的。只不过，从古至今，没有哪个社会也没有哪种制度是全无问题的，想要认真生活的人总是要思考到底什么才是自己真正想要去做的事，毕竟很少有人有能力改变世界，最多只是多多认识世界而已。

如果允许我再多写点的话，从我的视角看，你的文字透露出来好像你的状态是很紧绷的，你看到的世界和我看到的世界很不一样。当然也有可能是因为我平时容易无脑傻乐，过于乐观，所以看到你会觉得你有点悲观。

**Reply by XiangyunHuang** [reply]
简而言之，我去的每个馆、每个展览都是宏大叙事开头，在X时代、X精神、X文化下，在中法关系下，等等。我写的不过是展览里回避的部分。换个视角看，或者从画家本人的角度看，再写了点我怎么看。

我对美术界了解得越多越绝望，不过，我对人还是充满希望，上周，我去看央美本科生毕业展了，有的画挺有意思，学生也很有趣，这就是希望。有的专家老师说，毕业是学生们想怎么画就怎么画，等以后毕了业，与社会互动，需要闯出一条生路。是实话，也是令人惋惜的话，你像林风眠、吴冠中他们，他们才不管市场如何，只是画自己的画，找到自己的方向。直白地说，老师专家在强调生存是第一位的。

**Reply by earfanfan** [reply]
我在武汉和上海看过的，和你描述的不太一样，武汉美术馆的钱贵荪展、上海美术馆的林风眠吴冠中展都有提及画家的生平经历，这几位都是活了很久的人物，展中文字却并没有刻意回避苦难，也没有刻意渲染苦难，我很好奇是不是因为北京既是文化资源中心，也是政治资源中心，才如此特别。看来这需要以后我亲自去看看，才能分辨。

感到绝望，或者充满希望，悲观或者乐观，都只是一种选择。以前我也看过一些描述过文革时期的书影音，比如（PS比较冷门你大概未曾听闻）礼平的《晚霞消失的时候》，陈凯歌写的《少年凯歌》，余华的《许三观卖血记》，去年看的《三体》电视剧，你提到的《霸王别姬》当然也是看过的，我自己感觉创作者们各自是不同视角，所选择的态度也有些不同。

“选择”其实也可能是与个人信念挂钩，我以前看迅哥儿的书提到叔本华说过一句话，大意是说可以看别人写的书，去向别人借答案，但是不要让自己的脑子成为别人思想的跑马场。从我的视角看，你的选择和你刚好看过的内容有关，而刚好我看过的内容又与你不同。不过，我看过的内容此刻还没能捋出一些清晰的东西，暂时还跑着别人思想的马，容我再仔细想想，找到自己的马，回头再来你这继续展开写写。

**Reply by XiangyunHuang** [reply]
除了一些美术馆，我之前也去了中国电影博物馆、中国现代文学馆、共产党历史博物馆，我发现，美术、影视、文学作品，都会被放在一个宏大叙事里，而实际上，自己去认真观看的时候，感受未必与叙事一致，甚至有冲突。

首先，我要说这些地方说的事件都是真的，至于说如何解读、推导，又得出什么结论，取决于站在什么立场、什么思想上。其次，我要说这些地方对我还是比较开眼界的，尤其是影视方面，虽然我看了不少影视，但是，我去博物馆之后，我看到自己的无知。最后，我想说的是，无论看到的是什么，都要跳出固有的圈圈，自己想想。最好，结合自己来看看，形成自己的三观，我现在努力地这样做，这样对自己比较有意义。

**Reply by earfanfan** [reply]
我相信你，也看得懂你的忠告，额，本周末我需要回去一趟，等下周末我来完整写出我想的吧。或者不写在这里，给你写信吧。
